src/qt/guiutil.cpp

```diff
@@ -357,6 +366,21 @@ bool isObscured(QWidget *w)
         && checkPoint(QPoint(w->width() / 2, w->height() / 2), w));
 }
 
+void bringToFront(QWidget *w)
+{
+#ifdef Q_OS_MAC
+    id app = objc_msgSend((id) objc_getClass("NSApplication"), sel_registerName("sharedApplication"));
+    objc_msgSend(app, sel_getUid("activateIgnoringOtherApps:"), YES);
+#endif
+
+    if (w) {
+        // activateWindow() (sometimes) helps with keyboard focus on Windows
+        w->activateWindow();
```

Comment:
According to [Qt docs](https://doc.qt.io/qt-5/qwidget.html#activateWindow)

> Note that the window must be visible, otherwise activateWindow() has no effect.

Therefore, the safer sequence may be:
```
w->showNormal();  // or w->show();
w->activateWindow();
w->raise();